### DISCUSSION TITLE
"Select similar" tool for sub editor

### DISCUSSION THREAD

**Discussion by Quartofel** [FakeFishGames/Barotrauma#7478] [Ideas and suggestions]
Add ability to select only items that are the same as the one currently selected.

If you ever wanted to mass edit lights on your sub you know how tiresome it is to ctrl+click every single one of them one by one. Such tool would improve sub-building by a lot IMO.

**Comment by MaaiG** [comment]
I would suggest the following option:
![Image 010 (1)](https://user-images.githubusercontent.com/93348499/143592890-609f9839-4c7b-4098-a27b-e73e3c8265cb.png)
